hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientRatis.java

```diff
@@ -295,46 +294,39 @@ private XceiverClientReply newWatchReply(
   }
 
   @Override
-  public XceiverClientReply watchForCommit(long index)
-      throws InterruptedException, ExecutionException, TimeoutException,
-      IOException {
+  public CompletableFuture<XceiverClientReply> watchForCommit(long index) {
```

Comment:
Shall we change this one to `watchForCommitAsync` without breaking compatibility?